packages/client/src/index.ts

```diff
@@ -2,6 +2,7 @@ export * from "./client"
 export * from "./re-exports"
 export * from "./descriptors"
 export { InvalidTxError } from "./tx"
+export type * from "./tx/types"
```

Comment:
Wait, I'm thinking it's better to specify which ones to take out. There's a good amount of types so it will be somewhat long, but it will prevent us from leaking unwanted types in a future.

Reply:
Yes, better. Let's avoid `*` that we might regret in the future